indico/modules/categories/controllers/display.py

```diff
@@ -567,6 +569,10 @@ def _process(self):
 
 
 class RHCategoryCalendarViewEvents(RHDisplayCategoryBase):
+    class GroupBy(Enum):
+        category = 1
+        location = 2
```

Comment:
```python
class GroupBy(IndicoStrEnum):
    category = auto()
    location = auto()
```

Reply:
Fixed in [6ffa0f9](https://github.com/indico/indico/pull/6110/commits/6ffa0f92dd8d56120b9cb3fcee140b2d1053ff15), and thanks again :)